ROADMAP.md

```diff
@@ -0,0 +1,33 @@
+# YDB Embedded UI Roadmap
+## Legend
+We use the following symbols as abbreviations:
+
+1. ㉕ - feature appeared in the Roadmap for 2025;
+1. ㉔ - feature appeared in the Roadmap for 2024;
+1. ✅ - feature has been released;
+1. 🚧 - feature is partially available and is under development;
+1. ❌ - feature has been refused;
+1. 🔥 - not yet released, but we are in rush.
+
+## Diagnostics
+
+1. 🚧 **Capacity diagnostics** – a set of signals on a database info tab to easily diagnose CPU or storage shortage
+1. 🚧 **Basic built-in charts** – support for charts display for most popular signals: resource consumption, requests and latency
+1. 🚧 **Display Table Hot Keys** – on a tab **Hot Keys** for a [column-oriented](https://ydb.tech/docs/en/concepts/datamodel/table#olap-data-types) and [row-oriented table](https://ydb.tech/docs/en/concepts/datamodel/table#row-orineted_table) display Hot Keys based on query statistics
+1. ㉔ **Network diagnostics** – display signals about cluster network performance and help to discover reasons for potential problems
+
+
+## Cluster Administration
```

Comment:
Here we also discussed database creation and deletion, separate page for PDisk with support for PDisk restart

Reply:
>discussed database creation and deletion
I'm not sure we want this to be in the roadmap until we decided what to do with the feature
> separate pdisk page
I don't want to include features that don't add value per se. pdisk restart adds value. a new page with pdisk looks more like a UX reorg to me